Qt/InstaDam/Test/app/testLabel.cpp

```diff
@@ -69,6 +69,9 @@ void TestSelect::testLabelSetOpacity() {
     generateData();
     myLabel->setOpacity(2);
     QCOMPARE(ritem->SelectItem::opacity(), .02);
+    QCOMPARE(eitem->SelectItem::opacity(), .02);
```

Comment:
2: Test incomplete.
This test does not cover the changes in FreeDrawErase class.

Reply:
The FreeDrawErase class does not have an opacity as it is never painted. The class is designed to hold brush strokes that have been removed from FreeDrawSelect classes for undo and redo purposes only. The data from FreeDrawErase instances are not saved beyond the exiting of the program.